Userland/Libraries/LibJS/Runtime/Array.h

```diff
@@ -21,11 +21,11 @@ class Array : public Object {
     JS_OBJECT(Array, Object);
 
 public:
-    static ThrowCompletionOr<Array*> create(Realm&, u64 length, Object* prototype = nullptr);
-    static Array* create_from(Realm&, Vector<Value> const&);
+    static ThrowCompletionOr<NonnullGCPtr<Array>> create(Realm&, u64 length, Object* prototype = nullptr);
```

Comment:
Technically this commit title should be `LibJS: Convert Array::create{,_from}() to NonnullGCPtr` right? (The comma is missing currently)

Reply:
yep, fixed :+1: